tests/legacy-cli/e2e/tests/basic/rebuild.ts

```diff
@@ -1,13 +1,13 @@
-import { waitForAnyProcessOutputToMatch, silentNg } from '../../utils/process';
-import { writeFile, writeMultipleFiles } from '../../utils/fs';
 import fetch from 'node-fetch';
-import { ngServe } from '../../utils/project';
 import { getGlobalVariable } from '../../utils/env';
+import { writeFile, writeMultipleFiles } from '../../utils/fs';
+import { silentNg, waitForAnyProcessOutputToMatch } from '../../utils/process';
+import { ngServe } from '../../utils/project';
 
 export default async function () {
   const esbuild = getGlobalVariable('argv')['esbuild'];
   const validBundleRegEx = esbuild ? /complete\./ : /Compiled successfully\./;
-  const lazyBundleRegEx = esbuild ? /lazy\.module/ : /lazy_module_ts\.js/;
+  const bundleGeneratedRegEx = esbuild ? /bundle generation complete/ : /Compiled successfully/;
```

Comment:
The original intent for this was to ensure a new chunk was being created for the added lazy module.
Can you bring back the name-based check but with `chunk` instead for the esbuild case?